src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Overview/Cells/DiaryOverviewDayTableViewCell.swift

```diff
@@ -141,93 +248,121 @@ class DiaryOverviewDayTableViewCell: UITableViewCell {
 
 			entryStackView.addArrangedSubview(imageView)
 			entryStackView.addArrangedSubview(entryLabelStackView)
+			encountersVisitsContainerStackView.addArrangedSubview(entryStackView)
 
-			encountersVisitsStackView.addArrangedSubview(entryStackView)
-		}
+			// Let's draw a seperator line between each entryStackView.
+			let separatorLine = UIView()
+			separatorLine.backgroundColor = .enaColor(for: .hairline)
+			separatorLine.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
not needed if added to a stackView it inherits that from the stackview

Reply:
removed it.